### ISSUE TITLE
How to show a cell with a different colour?

### ISSUE THREAD

**Issue by bernardocout** [twinssbc/Ionic-Calendar#107]
Hi! 

I need to change the cell colour to show if a person is available or unavailable that day, how can I do this without modify the calendar's core?

Thanks in advance!

**Comment by twinssbc** [comment]
@bernardocout The calendar supports view customisation. You can get more information from README.

**Comment by bernardocout** [comment]
Yes, I was trying to change with view customization but I couldn't. I figured it out that I need to change the month template, specifically the behavior of getHighlightClass. Is there any way to do that?

**Comment by twinssbc** [comment]
@bernardocout YOu don't need to modify the source code. You can write your own template html file and pass it to monthviewDisplayEventTemplateUrl option.

**Comment by Aswin7955** [comment]
how can i give custom colour to the events? is there any options available like colour or event colour?

**Comment by twinssbc** [comment]
@Aswin7955 The calendar doesn't provide the color related options. You can write your own template and use ngClass to apply different color to different event types.
